### PR TITLE
Fix the Issue where user can click element multiple times before the page is fully loaded

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/mdm/src/mdm/api/user-api.jag [hunk 1]
```diff
@@ -96,13 +95,17 @@ if (uriMatcher.match("/{context}/api/user/login/")) {
...
-        try {
-            result = userModule.addUser(username, firstname, lastname, emailAddress, userRoles);
-        } catch (e) {
-            log.error("Exception occurred while trying to add a user to MDM User Store", e);
-            // http status code 400 refers to - Bad request.
-            result = 400;
+        if (username.length < mdmProps.usernameLength) {
+            log.error("Username Must be between 1 and " + mdmPops.usernameLength + " characters long");
+            result = "Username Must be between 1 and " + mdmPops.usernameLength + " characters long";
+        }else {
```
Comment (prabathabey) [inline modules/apps/jaggery/mdm/src/mdm/api/user-api.jag]: Formatting issues. Please fix

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/api/user-api.jag]: fix is committed along with the jira fixes

File: modules/apps/jaggery/mdm/src/mdm/units/policy-create/public/js/policy-create.js [hunk 2]
```diff
@@ -1897,10 +1897,74 @@ var showHideHelpText = function (addFormContainer) {
     }
 };
 
+function formatRepo (user) {
+    if (user.loading) {
+        return user.text
+    }
+    if (!user.username){
```
Comment (harshanL) [inline modules/apps/jaggery/mdm/src/mdm/units/policy-create/public/js/policy-create.js]: Check the formatting here.

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/units/policy-create/public/js/policy-create.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

File: modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js [hunk 3]
```diff
@@ -56,20 +57,22 @@ var backendServiceInvoker = function () {
...
-        if (accessToken){
-            return execute();
-        }else {
-            tokenUtil.refreshToken();
-            return execute();
+        if (accessToken) {
+            return execute(1);
+        } else {
+            var dummyRespose = {};
+            dummyRespose.status = 401;
```
Comment (prabathabey) [inline modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js]: Typo in "dummyRespose"

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js]: Noted.
The dummy Response was removed after the IS version bump.
Current "backend-service-invoker.js" doesn't have the above code snippet.

**Pull request merged**